GlotPress/GlotPress-WP
# Highlight leading and trailing spaces, and double/multiple spaces in the middle

## What?
Highlight the leading and trailing spaces both in the preview and editor rows
Also highlight double and multiple spaces in the middle.

This PR also adds a missing unit test for the invisible characters markup added on [prepare_original()](https://github.com/GlotPress/GlotPress/blob/develop/gp-templates/helper-functions.php#L17) for line breaks and tabs.

## Why?
Allows the translator to see the spaces at the beginning or the end of an original/translation.
Allows authors and translators to see double and multiple spaces in the middle of strings.

## How?
Wrap the spaces in a `<span>` with `invisible-spaces` class, which adds a grey color.

## Testing Instructions




1. Have some original string with spaces in the beginning or end, or with double spaces in the middle
2. Open the translation set where the string appears, see the highlighted spaces in grey in the preview row, both in original and preview (since https://github.com/GlotPress/GlotPress/pull/1500/commits/7a31b0fcab7db6b2502add1d52cec711fa08168a)
3. Open the translation string editor and see the highlighted spaces in grey, both in the original and translation.

![imagem](https://user-images.githubusercontent.com/7371591/194072868-a6b4acc3-7ce7-49ab-bcd6-e2edb115a532.png)

![imagem](https://user-images.githubusercontent.com/7371591/194073238-e75840c6-14a4-47fd-8d1a-bdfbe90efba8.png)

Fixes #863.
